### PR TITLE
add similar articles in bottom of each blog post

### PR DESCRIPTION
**Pull request by B2o5T**
related https://github.com/the-guild-org/the-guild-website/issues/603
fixes https://github.com/the-guild-org/the-guild-website/issues/546

- [x] Remove `twin.macro` that uses TailwindCSS v2
- [x] Intall  `tailwindcss` v3, `autoprefixer` and `postcss`
- [x] Install `clsx` package for concatenating classes
> `clsx` is faster than `classnames` [in 1.5 times](https://www.measurethat.net/Benchmarks/Show/7642/2/classnames-vs-clsx-vs-alternatives)

- [x] Add official prettier plugin `prettier-plugin-tailwindcss` for sorting classes
- [x] Add ESLint plugin `eslint-plugin-tailwindcss` and extend `recommended` config,  mostly for catching cases when unknown classes were used
- [x] Add `Prettier Check` GitHub action for verifying code formatting
- [x] Remove `styled-components`, `@types/styled-components`, and `babel-plugin-styled-components`
- [x] Refactor for using `stitches`

### CODE REVIEW

File: patches/@theguild+components+1.10.0.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@theguild/components/dist/index.js b/node_modules/@theguild/components/dist/index.js
+index 76a6979..6ef5653 100644
+--- a/node_modules/@theguild/components/dist/index.js
++++ b/node_modules/@theguild/components/dist/index.js
+@@ -2718,7 +2718,7 @@ var Navigation$1 = _styled__default["default"].nav(function (_ref) {
+     "right": "0px",
+     "bottom": "0px",
+     "left": "0px"
+-  }, react.css(_templateObject3$6 || (_templateObject3$6 = _taggedTemplateLiteral(["\n          z-index: 300; // TODO: Used for Docusaurus, remove when no longer needed.\n        "]))), !isModalOpen && react.css(_templateObject4$5 || (_templateObject4$5 = _taggedTemplateLiteral(["\n            top: -100vh;\n            bottom: 100vh;\n          "])))])];
++  }, react.css(_templateObject3$6 || (_templateObject3$6 = _taggedTemplateLiteral(["\n          z-index: 300; // TODO: Used for Docusaurus, remove when no longer needed.\n        "]))), !isModalOpen && react.css(_templateObject4$5 || (_templateObject4$5 = _taggedTemplateLiteral(["\n            top: -100%;\n            bottom: 100vh;\n          "])))])];
```
Comment (B2o5T) [inline patches/@theguild+components+1.10.0.patch:10]: saw some bug on mobile with `top: -100vh`, `top: -100%` fix it, need to fix on guild/components also

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2017",
+    "target": "ESNext",
```
Comment (B2o5T) [inline tsconfig.json:3]: allow use top-level await

**Review by n1ru4l — CHANGES_REQUESTED** [review]
image

I guess we should avoid referencing the same article? (https://the-guild-website-i7zg9djyk-theguild.vercel.app/blog/graphql-error-handling-with-fp)

**Review by n1ru4l — APPROVED** [review]

**Comment by n1ru4l** [comment]
@B2o5T Feel free to merge when you feel ready!

**Comment by B2o5T** [comment]
@n1ru4l it's ready :) just wanted to receive some other approves

**Review by charlypoly — APPROVED** [review]
🚀 🚀 🚀

**Pull request merged**